gcs/CHANGES.md

```diff
@@ -1,10 +1,22 @@
-### 2.2.8 - 2022-XX-XX
+### 2.2.8 - 2022-09-12
```

Comment:
Let's put all the changes 

like 
https://github.com/GoogleCloudDataproc/hadoop-connectors/pull/802
https://github.com/GoogleCloudDataproc/hadoop-connectors/pull/858

Reply:
Done